### DISCUSSION TITLE
zoom effect on text

### DISCUSSION THREAD

**Discussion by vdegenne** [3b1b/manim#1493] [Q&A]
I am really novice in both Python and Math, I think what I need to do requires a little bit of understanding of the library and Math transformation so I am asking help here. Because All I need is this simple effect really.

I already changed the `run_time` property in the `Write` class to slow down the animation when the text is being displayed.
What I would like to add is a little zoom effect. As the text slowly appears I would like, at the same time, add a zoom in effect. What I mean is the text should slowly appear a little bit bigger as it appears. 

Is that hard to implement ?

Hope to get help 🙏 Thanks

**Comment by TonyCrane** [comment]
You can try `turn_animation_into_updater(...)`
